src/OpenTelemetry.Instrumentation.Owin/Implementation/DiagnosticsMiddleware.cs

```diff
@@ -199,11 +210,37 @@ private static void RequestEnd(IOwinContext owinContext, Exception exception)
 
             activity.Stop();
 
+            if (OwinInstrumentationMetrics.HttpClientDuration.Enabled)
+            {
+                var tags = new TagList
```

Comment:
for up to 3 tags, its faster to directly pass them without TagList.
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/metrics (The doc probably need to call this out explicitly)

Reply:
It's mentioned in the doc under Best Practices: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/metrics#:~:text=When%20emitting%20metrics%20with%20more%20than%20three%20tags%2C%20use%20TagList%20for%20better%20performance.

> When emitting metrics with more than three tags, use `TagList` for better performance.

Reply:
@utpilla The missing part in doc is:
Do not use TagList for <3 tags, as that will have poorer performance than directly passing.